types/d3-cloud/index.d.ts

```diff
@@ -5,6 +5,8 @@
 
 import * as d3 from 'd3';
 
+export default d3.layout.cloud;
```

Comment:
It seems like the [underlying library](https://github.com/jasondavies/d3-cloud/blob/master/index.js#L10) is not using `export default`, so this .d.ts file should not either. Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.

Reply:
So the export should be like this?

```suggestion
export = d3.layout.cloud;
```